.github/workflows/cd-binaries.yaml

```diff
@@ -62,7 +64,7 @@ jobs:
       - run: |
           # fail fast if the release does not exist
           gh release view v${{ needs.prepare.outputs.release-name }} || exit 1
-          gh release upload v${{ needs.prepare.outputs.release-name }} *.txt *.zip *.deb *.rpm
+          gh release upload v${{ needs.prepare.outputs.release-name }} *.txt *.zip *.deb *.rpm *.json
```

Comment:
Should we be more specific of the json file we are uploading? The zip, rpm, and deb files are all clearly built things, but a JSON file could be used elsewhere (ex: some kind of configuration file). It seems this is scoped to output from the build steps though so maybe its ok.

Reply:
Matching the `openapi-*.json` pattern might be good so that it documents which file we are expecting to upload.